guide/getting-started/getting-started.md

```diff
@@ -34,70 +34,68 @@ Illustration sources
 
 # Getting started
 
-Welcome to the Bitcoin Design Guide! You’re in for an adventure. This chapter is all about the big concepts that are helpful to be familiar with when creating Bitcoin products. You’ll find a summary of concepts on this page, click through if you would like to dig deeper into a specific topic. The guide isn't a comprehensive overview of all Bitcoin topics, but only the ones you should know as a designer working in this space.
-
 ---
 
 ### [Why Bitcoin is unique]({{ '/guide/getting-started/why-bitcoin-is-unique/' | relative_url }})
 
-The internet has opened an infinite design space for information and communication. Bitcoin adds the new dimension of transferring value that could bring similar change as previous waves of innovation.
+Familiarize yourself with some of the most significant concepts that bitcoin designers need to know.
 
 ---
 
 ### [Why design for Bitcoin]({{ '/guide/getting-started/why-design-for-bitcoin/' | relative_url }})
 
-Bitcoin is fundamentally open and participatory and allows designers and creators to explore and shape experiences for users around the globe.
+Bitcoin is open-source, decentralized, permissionless, fungible, stateless, and non-custodial. If those words don’t mean anything to you, consider starting here.  
 
 ---
 
 ### [Open design]({{ '/guide/getting-started/open-design/' | relative_url }})
 
-Open design is about global collaboration that balances the free exchange of design information, the independence of individual designers and the collective power of collaboration.
+Bitcoin isn’t just changing money—it’s changing how designers collaborate across broad use cases. While open-source development has been around for years, open design is brand new.
 
 ---
 
 ### [The visual language of Bitcoin]({{ '/guide/getting-started/visual-language/' | relative_url }})
 
-As an open and public protocol, Bitcoin does not have a strictly defined brand. Yet there is a recognizable symbol that has evolved and remixed endlessly.
+Since bitcoin is public, open, and dependent upon no single person or organization, its brand is still undefined—and it might always be. But there’s still a to learn about how best to represent it visually.
```

Comment:
```suggestion
Since bitcoin is public, open, and dependent upon no single person or organization, its brand is still undefined—and it might always be. But there’s still a lot to learn about how best to represent it visually.
```